app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Not vaccinated")
+        elif visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("Outdated vaccine")
+        elif not visitor["wearing_a_mask"]:
+            raise NotWearingMaskError("Not wearing mask")
+        else:
+            return f"Welcome to {self.name}"

```

Comment:
The `else` statement here is unnecessary because the preceding `if` and `elif` statements already return or raise exceptions. You can directly return the welcome message without using `else`, which improves code readability as per the checklist guidelines.